### DISCUSSION TITLE
Can I haz Jaxlib release?

### DISCUSSION THREAD

**Discussion by dionhaefner** [jax-ml/jax#4245] [General] [answered]
Now that there's a tag for jaxlib 0.1.55, could it be possible to get a proper release, too?

We are kind of [waiting for one in mpi4jax](https://github.com/PhilipVinc/mpi4jax/pull/2), so we can use https://github.com/google/jax/pull/3834. Considering the last release was in July, it might be about time? :)

**Comment by jakevdp** [comment, marked as answer]
There is a new release in the works; see also https://github.com/google/jax/issues/4233.

**Reply by dionhaefner** [reply]
Sounds good! Sorry for the noise.

**Comment by zhangqiaorjc** [comment]
Thanks for the patience!

jaxlib-v0.1.55 has just been released.

https://github.com/google/jax/releases/tag/jaxlib-v0.1.55

**Comment by mattjj** [comment]
No noise complaints here; we appreciate the ping about it!

**Comment by PhilipVinc** [comment]
Thanks a lot @mattjj
